sample-runner/limestone/src/EnactImporter.js

```diff
@@ -5,8 +5,10 @@ import kind from '@enact/core/kind';
 import ActionGuide from '@enact/limestone/ActionGuide';
 import BodyText from '@enact/limestone/BodyText';
 import Button from '@enact/limestone/Button';
+import Card from '@enact/limestone/Card';
 import Checkbox from '@enact/limestone/Checkbox';
 import CheckboxItem from '@enact/limestone/CheckboxItem';
+import Chips from '@enact/limestone/Chips';
```

Comment:
We need to import Chip too.
```suggestion
import Chips, {Chip} from '@enact/limestone/Chips';
```